app/design/frontend/Slotin/SlotinTheme/Magento_Catalog/web/template/m2kocomponent-template.html

```diff
@@ -0,0 +1,4 @@
+<div class="component-wrapper">
+    <div data-bind="text: 'KnockOut Catalog Timer'"></div>
```

Comment:
Try using translations here, if this text is not an observable:
```
<div data-bind="i18n: 'KnockOut Catalog Timer'"></div>
```

Reply:
Added [translation](https://github.com/slotin/magento-slotin/pull/5/commits/866ff4598a5a517b06fd630458147681da29dd79)